doc/src/records/top-members.md

```diff
@@ -81,6 +81,19 @@ Number of subhour steps used per hour in the simulation. 4 is the time-honored v
   required: "No",
   variability: "constant") %>
 
+**nSubhrTicks=*int***
+
+Number of subhour ticks used per *nSubSteps* in the simulation.
+
+*Warning:* nSubhrTicks is in a experimental phase.
```

Comment:
"an" experimental phase

Reply:
"phase" doesn't really feel like the right term. Instead:

Note: This input is currently used only for experimental purposes.

Reply:
I made those changes as well as changed the warning description when using nSubHrTicks.